show/kube.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python -u
+# -*- coding: utf-8 -*-
+    
+from show.main import AliasedGroup, cli, run_command
+import os
+from swsssdk import ConfigDBConnector
+
+KUBECTL_CMD = "kubectl --kubeconfig /etc/sonic/kube_admin.conf"
```

Comment:
>/etc/sonic/kube_admin.conf [](start = 36, length = 26)

What happens if this file is missing?

Reply:
Good catch. Thanks. Code updated to handle.